README.md

```diff
@@ -1,4 +1,4 @@
-# Todo-list
+# Math Magicians
 This is a simple math magic application built with JavaScript, HTML, and CSS. It allows users to create a list of tasks with a description, completion status, and index.
 
```

Comment:
- [ ] Please always make sure that your `README` file is professional and that there is no redundancy in your document, but your `README` file has some redundancy like your tile and description are already added below the **Table of Content**. Kindly remove it from here.